packages/eslint-plugin/tests/rules/explicit-member-accessibility.test.ts

```diff
@@ -258,6 +258,21 @@ class Test {
         },
       ],
     },
+    {
+      filename: 'test.ts',
+      code: `
+class Test {
+  getX () {
```

Comment:
for sanity checking (it'll probably pass out of the box, but I'd like to guard against regressions), could you make this test the various permutations of modifiers:

- `public getX()`
- `public static getX()`
- `get getX()`
- `getX()`

Reply:
I haven't thought about theses cases, done!